src/style.css

```
@@ -0,0 +1,145 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px;
+  padding-bottom: 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 400px;
+}
+
+.card__img {
+  margin-left: 3px;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  height: 36px;
+  margin-bottom: 4px;
+  text-decoration: none;
+  color: #060b35;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-weight: normal;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__response {
+  margin-bottom: 24px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__stars-4 {
+  display: flex;
+}
+
+.card__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  flex-grow: 0;
+}
+
+.card__stars-1 > :nth-child(-n+1),
+.card__stars-2 > :nth-child(-n+2),
+.card__stars-3 > :nth-child(-n+3),
+.card__stars-4 > :nth-child(-n+4),
+.card__stars-5 > :nth-child(n) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__response-count {
+  font-size: 10px;
+  line-height: 14px;
+  flex-grow: 0;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  height: 18px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price-title {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  display: flex;
+  /* align-items: center; */
```

```suggestion
  /* align-items: center; */
```
Не залишай коменти. Але align-items: center можна було б застосувати для центрування контенту